ui/package.json

```diff
@@ -7,7 +7,8 @@
     "clean": "rimraf ./build/",
     "lint": "eslint '**/*.{js,ts,tsx,json}' && echo '💫  Lint complete.'",
     "lint:fix": "eslint '**/*.{js,ts,tsx,json}' --fix && npx sort-package-json  && echo '🛠  Lint --fix complete'",
-    "start": "webpack serve --hot --mode development"
+    "start": "webpack serve --hot --mode development",
+    "test": "jest"
```

Comment:
Might also be worth adding a something for `jest --watch`, so tests can re-run on save. Or I guess you can do `npm test --watch` if that's what you want.

Reply:
Yeah, `npm test -- --watch` can do this. I'll update the README with a note about this.